daemon/src/model/cfd.rs

```diff
@@ -345,6 +345,18 @@ impl fmt::Display for CfdState {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum UpdateCfdProposal {
+    Settlement {
+        proposal: SettlementProposal,
+        direction: SettlementKind,
+    },
+    RollOverProposal {
+        proposal: RollOverProposal,
+        direction: SettlementKind,
```

Comment:
If you reuse `SettlementKind` here, I guess it should be renamed to something more fitting?